pocs/camera/camera.py

```diff
@@ -17,6 +19,9 @@ def __init__(self,
                  model='simulator',
                  port=None,
                  primary=False,
+                 set_point=None,
```

Comment:
It looks like nothing happens with this `set_point` here. The abstract class already accepts the `kwargs` so I think it might be slightly confusing to specify it here and then do nothing with it.